Dockerfile

```
@@ -5,10 +5,10 @@
 # since node does not provide a buster image.
 # Added benefit of splitting the image
 # We copy the executable later
-FROM debian:bookworm AS json2capnpbuild
+FROM rust:slim-bookworm AS json2capnpbuild
 WORKDIR /app/services/json2capnp
 COPY services/json2capnp ./
-RUN apt-get update && apt-get -y --no-install-recommends install cargo ca-certificates
+RUN apt-get update && apt-get -y --no-install-recommends install ca-certificates
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)